sim/hunter/pet.go

```diff
@@ -277,7 +278,7 @@ func (hunter *Hunter) makeStatInheritance() core.PetStatInheritance {
 		return stats.Stats{
 			stats.Stamina:     ownerStats[stats.Stamina] * 0.3,
 			stats.Armor:       ownerStats[stats.Armor] * 0.35,
-			stats.AttackPower: ownerStats[stats.RangedAttackPower] * 0.22,
+			stats.AttackPower: ownerStats[stats.RangedAttackPower] * 0.22 * core.TernaryFloat64(hunter.HasPredatorArmor[3], 1.2, 1.0),
```

Comment:
Along similar lines, we could potentially create a function for updating stat inheritance, then in the set bonus code grab the old inheritance and return new inheritance multiplying the old value by the 1.2 multiplier